http-api-monitor-offers.adoc

```diff
@@ -0,0 +1,327 @@
+= Monitoring Bisq offers with Http API
+
+This guide walks you through the process of creating a bot that monitors available offers and sends email notifications.
+
+== What you'll build
+
+You'll build a NodeJS-based script that connects to Bisq over an HTTP API to get offers and market prices and sends email notification whenever "interesting" offers are detected.
+
+== What you’ll need
+
+* About 15 minutes
+* A favorite text editor or IDE
+* NodeJS 6+
+* Docker (to run from an image) or Git and Maven (to build from source)
+* A GMail account
+
+== Run the API
+
+There are two alternative ways you can run an instance of the Bisq HTTP API, either from a Docker image or from source.
+
+=== Run the API using Docker
+
+The easiest way to run the API in headless mode is by using a Docker image:
+
+    docker run -it --rm -p 8080:8080 -e BISQ_API_HOST=0.0.0.0 bisq/api
+
+=== Run the API from source
+
+For more hard-core developers that want to run from source:
+
+    git clone https://github.com/mrosseel/bisq-api
+    cd bisq-api
+    mvn compile exec:java \
+        -Dexec.mainClass="io.bisq.api.app.ApiMain" \
+        -Dexec.args="--appName=bisqmon"
```

Comment:
As I mentioned in 24ffd06, the `--appName` option and arg should be carried over to the `docker` arrangement as well. I have not done that in my edits.